server/data/incentivesApi.ts

```diff
@@ -40,9 +40,9 @@ class IncentivesApi extends RestClient {
   }
 
   async getLocationSummary(agencyId: string, locationPrefix: string): Promise<IncentivesLocationSummary> {
-    return this.get({
+    return (await this.get({
```

Comment:
you don't actually need to do this. because the function is async so is awaited elsewhere, adding await just before a return means there's a double-await on the same promise. this explains it better than i do: https://eslint.org/docs/rules/no-return-await

alternatively, adding the new await keyword means that the type assertion at the end ought to simply be `IncentivesLocationSummary` because the promise has already been resolved

Reply:
@ushkarev I've removed the async from this and bubbled up the Promises